src/libxdata/XData.cpp

```diff
@@ -66,7 +67,21 @@ void XData::cancelLoading() {
 }
 
 void XData::loadAirports() {
-    AirportLoader loader(world);
+    const AirportLoader loader(world);
+
+    //Custom scenery
+    CustomSceneryParser parser(xplaneRoot + "Custom Scenery/scenery_packs.ini");

```

Comment:
As mentioned in the general comment, this block should be in its own function that can be called from here inside a try block since it's legal to have no ini file at this point.

Reply:
I moved the custom scenery to another function and surrounded it with a try catch. I tested it by removing the scenery_packs.ini file and it loads the default scenery as expected.